shell/bashrc

```diff
@@ -319,7 +324,6 @@ function phpbrew ()
             unset PHPBREW_PATH
             eval `$BIN env`
             __phpbrew_set_path
-            echo "phpbrew is turned off."
```

Comment:
> another option is we can add a new command for iteratal operations with shell commands support, and keep `each` syntax. but I quite doubt if there is someone use `each` with phpbrew commands only

It's not only about that. It only supports the commands implemented in PHP because the command itself is currently implemented in PHP. I.e. it doesn't support the `fpm` command. Thus, I don't see much value in having two commands to support BC.

> i.e. `phpbrew each ext install xdebug` would definately fail on either 5.3 or 7.0

It's more of a made up example but yes, it will fail. I really don't know which internal commands users use in conjunction with `each`. The only command I'd use if it worked is `phpbrew each fpm {start|restart}` but it doesn't.